homeassistant/components/zwave_js/cover.py

```diff
@@ -79,17 +79,19 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
 
     async def async_open_cover(self, **kwargs: Any) -> None:
         """Open the cover."""
-        target_value = self.get_zwave_value("Open")
-        await self.info.node.async_set_value(target_value, PRESS_BUTTON)
+        target_value = self.get_zwave_value("targetValue")
```

Comment:
@kpine And is it also ok to use target value to open and close the device?

Reply:
As I understand it:
* Up/Open + True, means Start Level Change Command Up, means Increase the Level. So setting the target value to 99% means increase the level.
* Down/Close True, means Start Level Change Command Down, means Decrease the Level. So setting the value to 99% means decrease the level.

So it seems they are nearly the same. One difference I notice in zwave-js, a Multilevel Set will refresh the current value after 5 seconds, Start Level does not. Refreshing not be desirable for some devices. There are also some small differences in how the commands are supervised (if supported), not sure if that's relevant.

If you wanted to preserve the current behavior, the code could select between Open and Up like it does in stop. That wouldn't fix the original bug this PR is referencing though because the dumps are missing the Up/Down values, but I'm having a hard time accepting that is correct.

Sorry, I think that's the best I can answer it, w/o having used one of these devices. If we want a better answer as to what the best practice is for window covers, it could be asked [here](https://github.com/zwave-js/node-zwave-js/discussions).

Reply:
@MartinHjelmare  where I can find : homeassistant/components/zwave_js/cover.py

I'm connect with Putty on hassio but I can not find, thank you.

Reply:
If you want to test this PR, the easiest way on a Supervisor installation may be to copy the whole `zwave_js` integration directory from the source of this PR to a directory `custom_components` in your configuration directory and run the integration as a custom integration.